images/alpine/jdk11/Dockerfile.alpine.jdk11-mvn3.6-mono5-nuget3.4-choco0.10

```diff
@@ -15,6 +15,8 @@ ENV ChocolateyInstall=/opt/chocolatey
 
 COPY common-scripts /scripts
 COPY --from=builder /usr/src/choco/build_output/chocolatey $ChocolateyInstall
+# see https://github.com/mono/mono/issues/12557
```

Comment:
Fix for error that @steve-todorov mentioned in https://github.com/strongbox/ci-build-images/pull/17

Reply:
Thanks for mentioning it in a comment! :)